app/javascript/controllers/campaign_controller.js

```diff
@@ -24,6 +24,11 @@ export default class extends Controller {
         campaign_id: this.campaignId
       }
     )
+    this.toggleMapSelector()
+  }
+
+  toggleMapSelector() {
+    this.mapSelectorTarget.classList.toggle("show")
   }
```

Comment:
This kind of thing could be extracted to a separate toggle controller and the toggle method added as another action on the map selector options themselves. Worth thinking about if we need this elsewhere.

Good example here: https://boringrails.com/articles/better-stimulus-controllers/#example-toggling-classes

Reply:
My plan was to do this for the token drawer next, extracting any common patterns, so I'll defintely take a look at the article when I do that. Thanks!